src/workerd/io/compatibility-date.capnp

```diff
@@ -838,4 +839,20 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $experimental;
   # Enables support for Python workflows.
   # This is still in development and may change in the future.
+
+  unsupportedProcessActualPlatform @96 :Bool
+      $compatEnableFlag("unsupported_process_actual_platform")
+      $experimental;
+  # By default, Workerd will always expose "linux" as the process.platform.
+  # This flag enables support for process.platform to expose the actual system platform.
+  # This is unsupported, as this feature will never ever be supported as non-experimental and is a
+  # temporary WPT test path only.
+
+  enableNodeJsProcessV2 @97 :Bool
+      $compatEnableFlag("enable_nodejs_process_v2")
+      $experimental;
```

Comment:
I think it's likely safe for this one to have a default enable date and likely a `impliedByAfterDate` so that it's just the default when `nodejs_compat` is enabled. Typically we'd select a date that is at least a few weeks out from when we expect this to hit production.

Reply:
I'd like to aim make this update once we've finished up the FS related (chdir etc as commented) and stdio support side here. Will post the stdio PR soon, then perhaps we can figure out a timeline for the FS related methods and get the dates together.